src/pages/articles/cplusplus/Introduction to C++/index.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
We require the folder structure to be in kebab-case. i.e. `introduction-to-cplusplus`